openjdk-8-jre/Makefile

```
@@ -0,0 +1,6 @@
+SHELL=/bin/bash -e -x -o pipefail
+
+VERSION=1.0.0
+all:
```

checkmake (0.3.2)

[warning] Required target "all" must be declared PHONY.

(minphony)